README.md

```diff
@@ -36,6 +36,70 @@ over 100,000 metric values per second with less than 0.2% overhead.
 
 # Building the OVIS / LDMS source code
 
+## Pre-built containers
+
+You may skip the building process and use the pre-built containers from the
+following list. Please follow the corresponding links for more information on
+the images.
+- [ovishpc/ldms-samp](https://hub.docker.com/r/ovishpc/ldms-samp):
+  a small image for 'sampler' daemons meant to be deployed on compute nodes.
```

Comment:
Capitalize 'a'

Reply:
It displayed as:
- ovis/ldms-samp: a small image ...

It looks weird to have a capitalized 'A' there. I'll keep the lower case unless you strongly disagree.

Reply:
I see. Using lowercase is fine.